script.js

```diff
@@ -0,0 +1,100 @@
+const API_KEY  = "ec19ad523e949b880b38dce4fe91d6fe"
+let url = `https://api.themoviedb.org/3/movie/now_playing?api_key=ec19ad523e949b880b38dce4fe91d6fe`
+let search_url = 'https://api.themoviedb.org/3/search/movie?query='
+
+const form = document.querySelector('#search-form')
+const search = document.querySelector('#search-input')
+const movieContainer = document.querySelector('#movies-grid')
+const loadMore = document.querySelector('#load-more-movies-btn') 
+const closeButton = document.querySelector("#close-search-btn")
+const home = document.querySelector("h1")
+
+populateMovieSpace(url)
+
+home.addEventListener('click', () => {
+    movieContainer.innerHTML = ''
+    search.value = ''
+    populateMovieSpace(url)
+})
+async function populateMovieSpace(url) {
+    const response = await fetch(url)
+    const data = await response.json()
+    data.results.forEach( (movie) => {
+        generateCard(movie);
+    })
+}
+
+function generateCard(movieAPI){
+    //create star
+    let star = document.createElement('span') 
+    let starContent = document.createTextNode("⭐️")
+    star.appendChild(starContent)
+    star.classList.add("star")
+
```

Comment:
recommend using a helper function to make code more DRY (for creating/appending elements)